tools/wdio/plugins/custom-commands.mjs

```diff
@@ -193,6 +193,33 @@ export default class CustomCommands {
         })
     })
 
+    /**
+     * Waits for the session replay feature to initialize and start recording.
+     */
+    browser.addCommand('waitForPreloadRecorder', async function () {
```

Comment:
How is this different from `waitForSessionReplayRecording`?

Reply:
replayRecording can happen if the recorder is triggered in either the inst or the agg, while preloadRecorder is specific to just the the recorder running in the inst.  Probably need to go through and consolidate some of these commands at some point

Reply:
Sorry, I should probably know this but is the feat aggregate not loaded early when we need to start recording before page load?

Reply:
nope, only the recorder file.  The agg still imports at load time

Reply:
https://github.com/newrelic/newrelic-browser-agent/pull/951/files#diff-46275fa8336f66585ff8b5c90b61f397c044b2930b5b29c947ccc4e5614eadfaR52-R56

Reply:
Ok thanks. This makes sense then.